test/controllers/trackers_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class TrackersControllerTest < ActionController::TestCase
+  setup do
+    @tracker = trackers(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:trackers)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create tracker" do
+    assert_difference('Tracker.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.